src/Test/NginxTestCase.php

```diff
@@ -17,32 +17,37 @@
 /**
  * A phpunit base class to write functional tests with NGINX.
  *
- * You can define a couple of constants in your phpunit to control how this
- * test behaves.
+ * You can define constants in your phpunit.xml to control how this test behaves.
+ * 
+ * Note that the WEB_SERVER_HOSTNAME must also match with what you have in your
+ * NGINX configuration file.
  *
  * To define constants in the phpunit file, use this syntax:
  * <php>
  *     <const name="NGINX_FILE" value="./tests/FOS/HttpCache/Tests/Functional/Fixtures/nginx/fos.conf" />
  * </php>
  *
- * NGINX_BINARY       Executable for NGINX. This can also be the full path
+ * NGINX_FILE           NGINX configuration file (required if not passed to setUp)
+ * WEB_SERVER_HOSTNAME  hostname where your application can be reached (required)
+ * NGINX_BINARY         Executable for NGINX. This can also be the full path
  *                      to the file if the binary is not automatically found
  *                      (default nginx)
- * NGINX_PORT         Test NGINX port to use (default 8088)
- * NGINX_FILE         NGINX configuration file (required if not passed to setUp)
- * NGINX_CACHE_PATH   NGINX configuration file (required if not passed to setUp)
+ * NGINX_PORT           Port NGINX is listening to (default 8088)
+ * NGINX_CACHE_PATH     directory to use for cache
+ *                      (default /tmp/foshttpcache-nginx)
```

Comment:
I've updated the doc. BTW: The default is not `/tmp/foshttpcache-nginx` but `sys_get_temp_dir()` + `foshttpcache-nginx` (this is valid also for Vanish)

I've added the explanation in the "Description" column but dunno if change also del "Default" Column.